components/brave_wallet_ui/utils/string-utils.ts

```diff
@@ -89,7 +89,7 @@ export const formatAsDouble = (value: string): string =>
   // Removes all characters except numbers, commas and decimals
   value.replace(/[^0-9.,]+/g, '')
 
-export const isValidateUrl = (url: string) => {
+export const isHttpsUrl = (url: string) => {
   const re = /^\s*https?:\/\//
```

Comment:
this regex passes on HTTP URLs as well, which i don't think is what we want. you can just check url.startsWith('https://')

Reply:
@diracdeltas Fixed here: [cb03d72](https://github.com/brave/brave-core/pull/23297/commits/cb03d72954122bfb857ff8bc88b8c9d327be49ff)